sources/packages/backend/apps/queue-consumers/src/queue-consumers.module.ts

```diff
@@ -23,18 +23,23 @@ import {
   PartTimeMSFAAProcessResponseIntegrationScheduler,
   SFASIntegrationScheduler,
   ATBCResponseIntegrationScheduler,
+  ProcessArchiveApplicationsScheduler,
 } from "./processors";
 import {
   DisbursementScheduleService,
   SequenceControlService,
   StudentRestrictionSharedService,
   WorkflowClientService,
   ZeebeModule,
+  ApplicationService,
```

Comment:
Can this service be at `queue-consumers` scope?
We have been adding to `@sims/services` services that potentially could be reused by multiple apps. This one seems to be a `queue-consumers` task for now.